panel_chemistry/pane/py3dmol_viewer.py

```diff
@@ -8,11 +8,14 @@
 import panel as pn
 import param
 import types
+from dataclasses import dataclass
 
 try:
     import py3Dmol
 except ModuleNotFoundError:
-    pass
+    @dataclass
```

Comment:
I've included this workaround here because I could not import the file (and therefore not the whole `pane` module) without py3Dmol installed.